docs/develop/Election_Record.md

```diff
@@ -0,0 +1,44 @@
+### Election Record
+
+The Election Record are the files required at the end of the election to verify the election and are intended to be posted publicly. There should not be any sensitive data (eg: no guardian private keys) in the record. These include the following:
```

Comment:
"The Election Record is the group of files"

"the election and" -> "the election, and"

Reply:
Agree with the first, but the comma is not needed since the second statement is not an independent clause.